runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -565,17 +565,12 @@ private String detailsHtml(HiscoreResult result, HiscoreSkill skill)
 					Skill requestedSkill = result.getSkill(skill);
 
 					String rank = (requestedSkill.getRank() == -1) ? "Unranked" : StackFormatter.formatNumber(requestedSkill.getRank());
-					String exp = (requestedSkill.getRank() == -1) ? "Unranked" : StackFormatter.formatNumber(requestedSkill.getExperience());
+					long true_exp = Math.max(0, requestedSkill.getExperience());
```

Comment:
Shouldn't `skill.getExperience()` always be >= 0?

Reply:
Yes but due to buggy API sometimes it can be -1

Reply:
Yes but the API is buggy and can return -1